README.md

```diff
@@ -24,6 +24,7 @@ Possible parameters:
 * `grid` - (true|false) Defaults to false. Set to true to enable the alternative grid view
 * `title` - (founder_name|title_and_owner_name) What to use on the title of the fundraisers. If not present, uses the fundraiser's title
 * `campaign` - Campaign slug to pull fundraisers from
+* `partner_slug` - Filter result data by this field.
```

Comment:
hey @zemlyanoialex just to keep the consistency with the app, I would suggest to use `field_partner_slug` for this param name, mostly because we also have an entity named Partner in the app and it could cause some confusion (even though we're phasing this entity out of the app).